### ISSUE TITLE
Create Reg Page for 2021 Summit

### ISSUE THREAD

**Issue by claredillon** [InnerSourceCommons/InnerSourceMarketing#126]

**Comment by claredillon** [comment]
Create Reg page like APAC 2020 Summit

**Comment by claredillon** [comment]
Our first attempt was to use EventYay as we did in the APAC 2020 summit. We wanted to do this because it is (a) open source and (b) has a lovely way of displaying speakers/sessions. However, we have run into some difficulties in getting the Paypal payment system up and running.

After a lot of work over the weekend, our choice today was to continue to try and de-bug EventYay, or move back to EventBrite. After discussing the pros and cons with Cristina and Daniel earlier - we made the decision to go with EventBrite. We will continue to work on the issue with EventYay team and see what we did wrong to help us next time.

Reasons why we decided to move back to EventBrite this time round:

- Using the Paypal integration throws an error when using Paypal, even though we have set it up identically to last time. Because we always run a very tight schedule for summit planning, we don't have the time to troubleshoot this with the EventYay team, this time round.
- We are experiencing some other issues with EventYay, e.g. The attendee registration custom form seems to reset at random intervals.
- EventBrite offers more flexibility with the registration form (e.g. Only recording T-shirt details from people who want T-shirts, and checkbox options allowing us to ask which of the sessions they intend to attend)
- The EventYay platform does not process refunds automatically, EventBrite does.
- EventBrite allows us to email registered attendees from the platform and schedule such updates in advance. With EventYay, every update requires us to download the list and mail using personal emails and BCC - a lot of room for error and a lot of extra admin!

- Using EventBrite also means consistent experience for those who have registered for the community calls throughout the year.

We hope to put this live later today. It won't be as pretty as EventYay's version- but we intend to add the Speaker/Schedule details to the website instead, which will give the added benefit of having it there in the future.
